docs/Using-Fleet/fleetctl-CLI.md

```diff
@@ -253,7 +253,8 @@ spec:
 
 ## Using fleetctl with an API-only user
 
-Now that `fleetctl` and the Fleet server is configured, it can be helpful to create an API-only user to use when running automated workflows. An API-only user can be given a role based on the abilities it needs. The default access level is `Observer`. For more information on roles, see the [user permissions documentation](./Permissions.md#user-permissions).
+When running automated workflows using the Fleet API, we recommend using the API key from a specially-created API-only user rather than using the API key of a regular user. A regular user's API key expires regularly for security purposes, requiring regular update. Meanwhile, an API-only user's key does not expire.
```

Comment:
```suggestion
When running automated workflows using the Fleet API, we recommend a regular user's API key rather than using the API key of a regular user. A regular user's API key expires frequently for security purposes, requiring routine updates. Meanwhile, an API-only user's key does not expire.
```

Reply:
+1 to @Desmi-Dizney suggestions